chef/roles/ceph-calamari.rb

```diff
@@ -1,5 +1,5 @@
 name "ceph-calamari"
 description "Ceph Calamari Server"
 run_list(
-        "recipe[ceph::calamari]"
+        "recipe[ceph::role_ceph_calamari]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.